crates/wasm_interp/src/main.rs

```diff
@@ -38,14 +40,12 @@ fn main() -> io::Result<()> {
 
     let wasm_file_to_run = Arg::new(WASM_FILE)
         .help("The .wasm file to run")
-        .allow_invalid_utf8(true)
         .required(true);
 
     let args_for_app = Arg::new(ARGS_FOR_APP)
-        .help("Arguments to pass into the WebAssembly app\ne.g. `roc_wasm_interp app.wasm -- 123 123.45`")
+        .help("Arguments to pass into the WebAssembly app\ne.g. `roc_wasm_interp app.wasm 123 123.45`")
         .multiple_values(true)
-        .takes_value(true)
-        .last(true);
+        .takes_value(true);
```

Comment:
`zig test` doesn't pass the `--` to the Wasm interpreter so I had to get rid of it.
Removing the `.last()` seemed to do the trick.
This makes use behave the same way as Wasmer.